pkg/systemtemplate/template.go

```diff
@@ -333,103 +333,6 @@ spec:
 
 {{- end }}
 
-{{- if .IsWindowsCluster}}
```

Comment:
Since we're no longer generating any resources using `.IsWindowsCluster` we should remove that field from the templating struct in `import.go` as well as simplify the parameter in the `SystemTemplate` function